setup.py

```diff
@@ -26,14 +26,14 @@
     long_description_content_type="text/x-rst; charset=UTF-8",
     long_description=readme + "\n\n" + history,
     author="The scikit-build team",
-    author_email="scikit-build@googlegroups.com",
+    author_email="jcfr@kitware.com;henryfs@princeton.edu;matt.mccormick@kitware.com",
```

Comment:
Is this actually supported? I know how to do it in PEP 621 config, not sure if this is right here though.

Reply:
Good catch, I add reviewed spec associated with `mailto:` URI scheme and assumed this would work.

After reviewing the documentation[^1], using `,` instead of `;` should be supported.

It turns out that by specifying the following:

```python
setup(
...
    author_email="jcfr@kitware.com, henryfs@princeton.edu, matt.mccormick@kitware.com",
...
)
```

it leads to a `METADATA` file with

```
Author-email: jcfr@kitware.com, henryfs@princeton.edu, matt.mccormick@kitware.com
```

but the HTML generated on PyPI doesn't include the other emails:

![image](https://user-images.githubusercontent.com/219043/208053435-c3663fb9-a881-4ced-b09e-9f59c4e93f52.png)

Source: https://test.pypi.org/project/jcfr-pypi-sandbox/






[^1]: https://packaging.python.org/en/latest/specifications/core-metadata/#author-email

Reply:
Now since the author email field is optional and specifying multiple email is not supported, I will remove it.